operator/v1/types_image_content_source_policy.go

```diff
@@ -0,0 +1,99 @@
+package v1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ImageContentSourcePolicy holds cluster-wide information about how to handle registry mirror rules.
+// When multiple policies are defined, the outcome of the behavior is defined on each field.
+//
+// +kubebuilder:storageversion
+// Compatibility level 1: Stable within a major release for a minimum of 12 months or 3 minor releases (whichever is longer).
```

Comment:
@sanchezl isn't this sentence auto-generated?

Reply:
This sentence is auto-generated because of Line13 was add `// +openshift:compatibility-gen:level=1` https://github.com/openshift/api/pull/874/files#diff-4b23e0f12658da74708dafcfadb5466438307b0b334691d0556d817633e6f826R13

Reply:
> @sanchezl isn't this sentence auto-generated?

@sanchezl PTAL if this looks good?